presto-spi/src/main/java/com/facebook/presto/spi/block/ByteArrayBlockBuilder.java

```diff
@@ -213,6 +213,12 @@ public BlockEncoding getEncoding()
     }
 
     @Override
+    public BlockBuilder resetNew(BlockBuilderStatus blockBuilderStatus)
```

Comment:
All of the `resetNew` implementations should be placed next the the existing `reset` method